main.go

```diff
@@ -137,11 +133,19 @@ func parseLabel(line string) (lbl string, rest string) {
 			ind := inds[0]
 			lbl = strings.TrimSpace(line[:ind])
 			rest = line[ind:]
-			return
 		}
 	}
 
-	return "", line
+	// Labels should not contain space
+	matched, err := regexp.MatchString(`\S*\s+\S*`, lbl)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
This error handling looks non-idiomatic. Do you think it would look better with "MustCompile", since we know that the regexp is valid?

```go
matched := regexp.MustCompile(`\S*\s+\S*`).MatchString(lbl)
```

Reply:
Yes that would be great! However, I accidentally deleted my fork, so I cannot edit this pull request anymore. Could you please edit this commit?

Reply:
Sure :) But apparently I have to recreate the PR for that, couldn't find another way